### PR TITLE
Add option to browse reddit to story

### PR DESCRIPTION
**Pull request by ckane30691**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2678)

### CODE REVIEW

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, I don't see the conflict. Could you check again?

File: english/marshmallow.md [hunk 1]
```diff
@@ -529,6 +529,8 @@ Click [here](young/young.md) to listen to music.
 
 [Attack it with a lightsaber](lightsaber/lightsaber.md)
 
+[Decide to browse Reddit instead](https://www.reddit.com/)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Looks good to me! :car:

**Pull request merged**